app/controllers/my_modules_controller.rb

```diff
@@ -277,7 +277,10 @@ def protocol
 
   def update_protocol
     protocol = @my_module.protocol
+    old_protocol_name = protocol.name
     protocol.update!(protocol_params)
+
+    log_activity(:protocol_name_in_task_edited) if old_protocol_name != protocol.name
```

Comment:
Should be wrapped in transaction, also you can probably use `protocol.name_changed?`